### PR TITLE
 117910807 add username for user

### PR DESCRIPTION
**Pull request by rustemmukhamadiev**

### CODE REVIEW

File: db/migrate/20160419143609_add_username_for_user.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUsernameForUser < ActiveRecord::Migration
+  def change
+    add_column :users, :username, :string, null: false, limit: 60, unique: true
```
Comment (vladimirbazhanov) [inline db/migrate/20160419143609_add_username_for_user.rb]: @rustemmukhamadiev why you want new field, full_name look like did the same? :)

Comment (rustemmukhamadiev) [inline db/migrate/20160419143609_add_username_for_user.rb]: @vladimirbazhanov  
Look like on specification https://flatstack.atlassian.net/wiki/pages/viewpage.action?pageId=22052887

As user I want to:
view 10 recent posts of specified user at /username URL (sorted by creation date)

Comment (vladimirbazhanov) [inline db/migrate/20160419143609_add_username_for_user.rb]: hmm. ok 👍

Comment (rustemmukhamadiev) [inline db/migrate/20160419143609_add_username_for_user.rb]: =) not need to redo the tests back

**Pull request merged**